src/Microsoft.Health.Fhir.Stu3.Api/Microsoft.Health.Fhir.Stu3.Api.csproj

```diff
@@ -8,7 +8,7 @@
   <ItemGroup>
     <PackageReference Include="Ensure.That" Version="8.1.2" />
     <PackageReference Include="FluentValidation" Version="8.6.1" />
-    <PackageReference Include="Hl7.Fhir.STU3" Version="1.3.0" />
+    <PackageReference Include="Hl7.Fhir.STU3" Version="1.4.0" />
```

Comment:
If I upgrade this to 1.5.0-beta1, 408 stu3 crucible tests are failing because of different reasons
https://microsofthealthoss.visualstudio.com/FhirServer/_apps/hub/ms.vss-releaseManagement-web.hub-explorer?releaseId=2104&_a=release-pipeline-progress

Reply:
Is it an issue that the stu3 packages are 1.4.0 while the common packages are 1.5.0-beta? Do we even want to be taking a beta version?

Reply:
Right, we are not going ahead with beta version. I am closing this PR.